airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/test/models.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+import os
+from typing import Dict, Set
+
+from pydantic import BaseModel, Field, validator
+
+
+class AirbyteCiPackageConfiguration(BaseModel):
```

Comment:
This isn't used as part of this PR, but it is used in [this follow up PR](https://github.com/airbytehq/airbyte/pull/34736) by [get_airbyte_ci_package_config](https://github.com/airbytehq/airbyte/pull/34736/files#diff-b86e6d2f27d70812919004f8b786a5bb0ea5a7a28ec99c5ffd8773d9eb424eecR85-R96), which is itself called by [the test command](https://github.com/airbytehq/airbyte/pull/34736/files#diff-b86e6d2f27d70812919004f8b786a5bb0ea5a7a28ec99c5ffd8773d9eb424eecR271) to identify the poe_tasks to run and how to configure the container :+1: